packages/emails/email-manager.ts

```diff
@@ -208,37 +226,42 @@ export const sendScheduledSeatsEmails = async (
 };
 
 export const sendCancelledSeatEmails = async (calEvent: CalendarEvent, cancelledAttendee: Person) => {
-  const clonedCalEvent = cloneDeep(calEvent);
+  const calendarEvent = formatCalEvent(calEvent);
```

Comment:
If we format it here, then on line 233 shouldn't we now pass "calendarEvent" that has been formatted? We are passing "calEvent" which would have platform info we want to hide, no?